### PR TITLE
Feature/tests pages specs utils

### PR DESCRIPTION
**Pull request by SvetlanaShardakova**
The first part of tests and specs was added.

### CODE REVIEW

File: tests/pageObject/loginTermsPage.page.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  get termsOfUseLink() {
+    return $('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(4) > div > label > a:nth-child(5)')
+  }
+
+  get privacyPolicyLink() {
+    return $('#root > div > div > div > div:nth-child(1) > div > div > div:nth-child(2) > div:nth-child(4) > div > label > a:nth-child(7)')
+  }
+
+  get confirmAndContinueButton() {
+    //return $('#root > div > div > div > div:nth-child(1) > div > div > div._17RbzQLX')
```
Comment (stcherenkov) [inline tests/pageObject/loginTermsPage.page.js]: Would be great to have a description comment about why do we have this commented selector

File: tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+// LoginTermsPage(Agreements) are opened
+describe('Initial app loading', () => {
+  beforeAll(() => {
+    loginTermsPage.open()
+  })
+
+  // '"Confirm and continue" button is available and leads to the next page'
+  // Step 1. Click to 'Continue' -> 'Continue' is disabled
+     it('Click to "Continue" -> "Continue" is disabled', () => {
```
Comment (stcherenkov) [inline tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js]: Something weird happened with indentations here

File: tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+     //Click to 2st and 4th checkboxes
+     loginTermsPage.checkboxes[1].click()
+     loginTermsPage.checkboxes[3].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[1].click()
+     loginTermsPage.checkboxes[3].click()
+
+     //Click to 3rd and 4th checkboxes
+      loginTermsPage.checkboxes[2].click()
```
Comment (stcherenkov) [inline tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js]: And here

File: tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      loginTermsPage.checkboxes[3].click()
+     expect(loginTermsPage.confirmAndContinueButton.getAttribute('disabled')).toBe('true')
+     loginTermsPage.checkboxes[2].click()
+     loginTermsPage.checkboxes[3].click()
+  })
+
+  //3. Check  any 3 checkboxes(for example - 1, 2, 4) and try to click -> 'Continue' is disabled
+  it('', () => {
```
Comment (stcherenkov) [inline tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js]: What are we testing here?

File: tests/specs/T476--StartPage-PageChecks.spec.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
+const loginTermsPage = require('../pageObject/loginTermsPage.page')
+const startPage = require('../pageObject/startPage.page')
+
+describe('Create or Import first wallet', () => {
+  beforeAll(() => {
+    loginTermsPage.open()
+    loginTermsPage.checkboxes.forEach((checkbox) => {
+        checkbox.$('label').click()
+    loginTermsPage.confirmAndContinueButton.click();
```
Comment (stcherenkov) [inline tests/specs/T476--StartPage-PageChecks.spec.js]: Weird indentations here too. Maybe we should set up some check-up or autocorrect tool for you?

File: tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js (hunk 4 above)
Comment (SvetlanaShardakova) [reply tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js]: We are checking that the 'Continue' button is disabled and it's not possible to proceed with only 3 checked checkboxes. Do you think that these tests are redundant?

File: tests/pageObject/loginTermsPage.page.js (hunk 1 above)
Comment (SvetlanaShardakova) [reply tests/pageObject/loginTermsPage.page.js]: This delimiter will be deleted at the next  PR, thanks!

File: tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js (hunk 4 above)
Comment (stcherenkov) [reply tests/specs/T433--TermsAndConditionsPage-AcceptAndContinue.spec.js]: There is nothing in test description, just empty quotes

**Pull request merged**